uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.js

```diff
@@ -379,13 +379,24 @@ define([
         }
 
         if (missingRequiredFields.length) {
-          var notification = CRM.alert(missingRequiredFields.join(', '),
-            missingRequiredFields.length === 1 ? 'Required field' : 'Required fields', 'error');
+          var notification;
+          var notificationTitle = missingRequiredFields.length === 1 ? 'Required field' : 'Required fields';
+          var missingFields = missingRequiredFields.join(', ');
+
+          if ($scope.isRole('manager')) {
+            $rootScope.$broadcast('ssp:notify', {
+              type: 'error',
+              title: notificationTitle,
+              body: missingFields
+            });
+          } else {
+            notification = CRM.alert(missingFields, notificationTitle);
```

Comment:
@bmnepali As in SSP we dont have `#crm-notification-container`, it might be easier to add that markup in ssp and style it, so that we dont need to use toaster.
cc @AkA84